src/lib/queue_worker.js

```diff
@@ -113,7 +113,7 @@ QueueWorker.prototype._resetTask = function(taskRef, deferred) {
       } else {
         var errorMsg = 'reset task errored too many times, no longer retrying';
         logger.debug(self._getLogEntry(errorMsg), error);
-        deferred.reject(errorMsg);
+        deferred.reject(new Error(errorMsg));
```

Comment:
Ah, sorry, was confused for a second between `reject` and `_reject` - this could be useful